ddp-workspace/projects/ddp-sdk/src/lib/services/address.service.ts

```diff
@@ -26,9 +30,17 @@ export class AddressService extends UserServiceAgent<Address> {
                 return new AddressVerificationResponse(data.body);
             }),
             catchError((error) => {
-                return throwError(error.error as AddressVerificationStatus);
-            })
-        );
+                const verificationStatus = error.error as AddressVerificationStatus;
+                return this.translate.getTranslation(verificationStatus.errors
+                    .map(addressError => ERROR_MSG_TRANS_KEY_PREFIX + addressError.code))
+                    .pipe(
+                        take(1),
+                        map(codeToMsg => {
+                            verificationStatus.errors = verificationStatus.errors.map(
+                                addressError => ({ ...addressError, message: codeToMsg[ERROR_MSG_TRANS_KEY_PREFIX + addressError.code] }));
+                            throw verificationStatus;
+                        }));
```

Comment:
Swapping error messages from server to local translation. Expect that this not not particularly helpful for English, but needed for other languages.